css-layout-classes/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Document</title>
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-full">
+        <header><h1>Choose your character!</h1></header>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-one-third">
+        <img src="images/link.png">
+      </div>
+      <div class="column-one-third">
+        <h2>Super Smash Bros. Ultimate</h2>
+        <p>Super Smash Bros. Ultimate received universal acclaim from both critics and players, with some critics calling it the best installment in the series. It received praise for it's large amount of content and fin-tuning of existing Smash gameplay eleemtns, although its online mode was widely criticized.</p>
+      </div>
+      <div class="column-one-third">
+        <img src="images/yoshi.png">
+      </div>
+    </div>
+   <div class="row">
+     <div class="column-one-fourth"><img src="images/kirby.png"></div>
+     <div class="column-one-fourth"><img src="images/fox.png"></div>
+     <div class="column-one-fourth"><img src="images/pikachu.png"></div>
+     <div class="column-one-fourth"><img src="images/ness.png"></div>
+   </div> 
+   
+    <div class="row">
+      <div class="two-thirds">
+        <h2>Donkey Kong</h2>
+        <p>Donkey Kong is a series of video games featuring the adventures of a gorilla called Donkey Kong, conceived by Shigeru Miyamoto in 1981. The franchise consists mainly of two game genres, but also includes additional spin-off titles of various genres.</p>
+        <p>The games of the first genre are mostly single-screen paltform/action puzzle types, featuring Donkey Kong as the opponent in an industrual construction setting.</p>
+      </div>
+      <div class="one-third">
+        <img src="images/dk.png">
+      </div>
+    </div>
+    <div class="row">
+      <div class="one-third">
+        <img src="images/mario.png">
+      </div>
+      <div class="two-thirds">
```

Comment:
These class names are incorrect as compared to your stylesheet, they should be `column-one-third` and `column-two-thirds`